lib/matplotlib/tests/test_quiver.py

```diff
@@ -42,32 +42,32 @@ def test_quiver_key_memory_leak():
 
 def test_quiver_number_of_args():
     X = [1, 2]
-    with pytest.raises(TypeError,
-            match='takes 2-5 positional arguments but 1 were given'):
+    with pytest.raises(
+        TypeError, match='takes 2-5 positional arguments but 1 were given'):
```

Comment:
I don't quite like this indentation because the continuation is at the same indent as the context block.

Reply:
sure, changed